TfL image processing UI/src/app/services/app-init.service.ts

```diff
@@ -20,10 +20,18 @@ export class AppInitService {
     }
 
     try {
-      // Business logic to inject the API key is for you to fill in.
       const headers = new HttpHeaders().set('Content-Type', 'text/plain; charset=utf-8');
-      const resolve = await http.get(server + 'GoogleMapsApiKey', { headers, responseType: 'text' }).toPromise();
-      googleMapsConfig.apiKey = resolve;
+
+      const uiProjectDeploymentId: string = "48366730-b6d7-4332-bd5e-3f4deb7c4d6a"; // prod deployment id
```

Comment:
I would just make this simpler, call GoogleMapsApiKey environment variable, and if there isnt one, set "". People can set it in Quix if they want.